packages/aws-cdk-lib/aws-secretsmanager/lib/rotation-schedule.ts

```diff
@@ -126,18 +128,31 @@ export class RotationSchedule extends Resource {
       );
     }
 
-    let automaticallyAfterDays: number | undefined = undefined;
-    if (props.automaticallyAfter && props.automaticallyAfter.toDays() > 1000) {
-      throw new Error(`automaticallyAfter must not be greater than 1000 days, got ${props.automaticallyAfter.toDays()} days`);
-    }
-    if (props.automaticallyAfter?.toMilliseconds() !== 0) {
-      automaticallyAfterDays = props.automaticallyAfter?.toDays() || 30;
+    let automaticallyAfterDays: number | undefined;
+    let scheduleExpression: string | undefined;
+    if (props.automaticallyAfter) {
+      if (props.automaticallyAfter.toMilliseconds() > 0) {
```

Comment:
`props.automaticallyAfter.toMilliseconds()` 

I think this is repeating and we can keep this as a variable.